ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -90,6 +108,27 @@ message CreatedEvent {
   ContractMetadata metadata = 10;
 }
 
+// View of a create event matched by an interface filter.
+message InterfaceView {
+
+  // The interface implemented by the matched event.
+  // Required
+  Identifier interface_id = 1;
+
+  // Whether the view was successfully computed, and if not,
+  // the reason for the error. The error is reported using the same rules
+  // for error codes and messages as the errors returned for API requests.
+  // Required
```

Comment:
So if the interface filter does not request the interface view, then this is always `OK`? If so, that may be worth documenting.

Reply:
@nmarton-da what actually happens her at the moment? I think `Optional` and not setting it might make more sense than setting it to `OK`.